infra/base-images/base-runner/test_all

```diff
@@ -24,12 +24,21 @@ for FUZZER_BINARY in $(find $OUT/ -executable -type f); do
   fi
 
   FUZZER=$(basename $FUZZER_BINARY)
-  if echo "$FUZZER" | grep "^afl-" > /dev/null 2>&1; then
+  if [[ "$FUZZER" == afl-* ]]; then
```

Comment:
hmm, how does this work? I thought afl-* would expand to all filenames that begin with afl- ?